docs/source/app_developers_guide/go_sdk_import.rst

```diff
@@ -0,0 +1,59 @@
+********************
+Importing the Go SDK
+********************
+
+.. note::
+   The Sawtooth Go SDK assumes that you have the latest version of Go,
+   more information on installing and configuring can be found at `go
+   <https://github.com/golang/go>`_.
+
+Once you've got a working version of Sawtooth, there are a few additional
+steps you'll need to take to get started developing for Sawtooth in Go.
+
+1. Get Go SDK located at `Sawtooth Go SDK repository
+   <https://github.com/hyperledger/sawtooth-sdk-go>`_.
+
+.. code-block:: ini
```

Comment:
'ini' is not the right lexer to chose here (or below). 'console' might be approprite. The list of lexers is available at:

http://pygments.org/docs/lexers/

Reply:
Good catch, thanks. Also below import section should have been 'go'.